package.json

```
@@ -56,7 +56,8 @@
   },
   "resolutions": {
     "**/prismjs": "1.27.0",
-    "**/react": "^17.0.0"
+    "**/react": "^18.2.0",
+    "**/@types/react": "^18.2.6"
```

🤔 Do we need to specify the exact minors for these or can simply pin to a major?
```suggestion
    "**/react": "^18.0.0",
    "**/@types/react": "^18.0.0"
```

---

Yeah that's a good point. `^` would still do its job without the need to updating these lines when we upgrade to a newer minor version but it's definitely cleaner to just specify major. I replaced both with `^18` 👍🏻